addon/components/paper-card-actions.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import FlexMixin from '../mixins/flex-mixin';
+
+const { Component, computed, String: Str } = Ember;
+
+export default Component.extend({
+  tagName: '',
```

Comment:
What was the reason for making this tagless?
You could make it have a name of `md-card-actions` and compute the needed classes here.